felix/bpf/nat/maps.go

```diff
@@ -101,11 +101,26 @@ var ZeroCIDR = ip.MustParseCIDROrIP("0.0.0.0/0").(ip.V4CIDR)
 
 type FrontendKey [frontendKeySize]byte
 
+type FrontendKeyInterface interface {
```

Comment:
Naming wise, I think symmetry between V4 and V6 would be good:
* FrontendKey = the interface
* FrontendKeyV4 = v4 version 
* FrontendKeyV6 = v6 version

Reply:
Ohhhh yeah, thought about it, then though about how much editing it would be :sweat_smile: